lib/parse.js

```diff
@@ -38,6 +38,7 @@ function Parse () {
   me._stream = new BlockStream(512)
   me.position = 0
   me._ended = false
+  me._hardLinks = []
```

Comment:
A `link -> target` map object would be a more efficient and useful data structure than an array.